pyoozie/client.py

```diff
@@ -505,6 +505,13 @@ def job_coordinator_resume(self, coordinator_id=None, name=None, user=None):
             return True
         return False
 
+    def job_coordinator_update(self, coordinator_id=None, name=None, user=None):
+        coord = self._fetch_coordinator_or_action(coordinator_id, name, user)
+        if coord.status.is_active() or coord.status.is_running() or coord.status.is_suspended():
```

Comment:
Isn't this highly redundant? `RUNNING` and `SUSPENDED` should both be covered by `is_active`.